Docs/sphinx/Ceptr.rst

```diff
@@ -19,13 +19,27 @@ To install CEPTR dependencies::
   $ cd ${PELE_PHYSICS_HOME}/Support/ceptr
   $ poetry update
 
+.. note:: Note that you will need to use a Python version >=3.10 and <3.12. If a compatible version exists in your system then poetry will try to find and use it. Otherwise, think about using a `conda <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_ environment to manage packages and their dependencies without tampering with your system. 
```

Comment:
How about saying to check Support/ceptr/pyproject.toml for the latest supported python versions in case they get updated and we forget to modify this text in the future?

Reply:
Done.